invokers/python/pyfunc/__main__.py

```diff
@@ -2,31 +2,36 @@
 # SPDX-License-Identifier: BSD-2-Clause
 
 from pyfunc import invoke, find_func
+from pyfunc.constants import *
 
 import sys
 from argparse import ArgumentParser
 
 
 def check(args):
     try:
-      find_func(args.search_path)
+      find_func(args.search_path, args.module_name, args.function_name)
     except Exception as e:
       print("Failed to detect function: " + str(e), file=sys.stderr)
       sys.exit(1)
 
 def start(args):
-    invoke.main(args.search_path)
+    invoke.main(args.search_path, args.module_name, args.function_name)
 
 parser = ArgumentParser(prog='pyfunc')
 parser.set_defaults(func=lambda: parser.print_help())
 subparsers = parser.add_subparsers(help='sub-command help')
 
 parser_check = subparsers.add_parser('check', help='check if the module and function can be loaded')
 parser_check.add_argument('-s', '--search_path', type=str, default='.')
```

Comment:
Should we update the default to `SEARCH_PATH_DEFAULT` here as well? Similarly for start. Otherwise we can do that in new PR

Reply:
Oops, just saw this. I don't think it's necessary because all the defaults are in one place now. See: https://github.com/vmware-tanzu/function-buildpacks-for-knative/pull/144#discussion_r998566338

Reply:
Oh good catch!

Reply:
I saw this too late but I'll do a short follow-up PR

Reply:
PR: https://github.com/vmware-tanzu/function-buildpacks-for-knative/pull/149